src/main/java/previewcode/backend/api/v1/WebhookAPI.java

```diff
@@ -94,7 +121,7 @@ private void handleNewPullRequest(JsonNode body) throws IOException {
         String token = JWT.create()
                 .withIssuedAt(now)
                 .withExpiresAt(exp)
-                .withIssuer(INTEGRATION_ID)
+                .withIssuer(TEST_INTEGRATION_ID)
```

Comment:
We really need to find a way to easily switch between 'production mode' on the server and 'test/development mode'. My first guess would be to do the switch based on an environmental variable. All things that differ between development and production can then be configured by the dependency injection system.